app/package.json

```diff
@@ -32,6 +32,7 @@
     "electron-spellchecker": "1.1.2",
     "electron-updater": "2.18.2",
     "electron-window-state": "4.1.1",
+    "is-online": "^7.0.0",
```

Comment:
Should bt the exact working version here.

Reply:
Done